dist/pinny.js

```diff
@@ -443,6 +445,19 @@
             return $(document.activeElement);
         },
 
+        /*
+        * In iOS, there is an issue when soft keyboard triggers, position fixed
+        * elements on screen don't get updated, distorting the UI.
+        * Fix: blur keyboard on orientationchange
+        */
+        _blurOnOrientationChange: function($pinny) {
```

Comment:
I feel like this method should be named more for what it fixes rather than what it does. This would eliminate the need for a super detailed comment, or the comment could explain why this fixes your issue.

http://blog.codinghorror.com/coding-without-comments/